contrib/IECoreMantra/src/IECoreMantra/procedural/World.cpp

```diff
@@ -73,7 +73,7 @@ class VRAY_ieWorld : public ProceduralPrimitive {
 	virtual int	  initialize(const UT_BoundingBox *);
 	virtual void	 getBoundingBox(UT_BoundingBox &box);
 	virtual void	 render();
-	UT_String m_worldFileName;
+	UT_StringHolder m_worldFileName;
```

Comment:
Shouldn't change the member types mid-version, so lets just put it in an H16 block if needed.